pyaedt/hfss.py

```diff
@@ -110,13 +110,13 @@ class Hfss(FieldAnalysis3D, object):
 
     """
 
-    def __repr__(self):
-        try:
-            return "HFSS {} {}. ProjectName:{} DesignName:{} ".format(
-                self._aedt_version, self.solution_type, self.project_name, self.design_name
-            )
-        except:
-            return "HFSS Module"
+    # def __repr__(self):
+    #     try:
+    #         return "HFSS {} {}. ProjectName:{} DesignName:{} ".format(
+    #             self._aedt_version, self.solution_type, self.project_name, self.design_name
+    #         )
+    #     except:
+    #         return "HFSS Module"
```

Comment:
@maxcapodi78 Do you think that we can get rid of the __repr__ method?

Reply:
It's actually making the code explode under certain conditions.